doc/manual/de/config/widgets/urltrigger/index.rst

```diff
@@ -0,0 +1,90 @@
+.. _urltrigger:
+
+Das UrlTrigger Widget
+=====================
+
+.. api-doc:: UrlTrigger
+
+Beschreibung
+------------
+
+Das UrlTrigger Widget fügt der Visualisierung eine Schaltfläche hinzu, die im Hintergrund Daten von einer
+URL abfragt.
```

Comment:
Ich finde die Formulierung nicht optimal, da ich unter einer Abfrage erwarte, dass das Ergebnis irgendwie präsentiert wird. Der URL Trigger macht aber nur im Hintergrund einen Aufruf. Also nur in eine Richtung gehend